tests/checkmk-docker-hooks/post-start/post-configure-checkmk.sh

```diff
@@ -4,9 +4,19 @@
 su - cmk -c "/usr/bin/env bash" << __EOF__
 
 # Watch for new rrd files and fill them with random data
-python3 -m pip install watchdog
-export GRRD_FORK=1
-python3 -u /docker-entrypoint.d/post-start/generate_random_rrd_data.py
+# python3 -m pip install watchdog
+# export GRRD_FORK=1
+# python3 -u /docker-entrypoint.d/post-start/generate_random_rrd_data.py
+
+# Wait until it is ready
+until omd status
+do
+    echo Wait until Checkmk is ready...
+    sleep 1
+done
+
+echo Waiting 20 more seconds just to be sure it can reply...
+sleep 20
```

Comment:
could we do a similar `until` loop here with a curl command to query the version endpoint of the rest-api?